modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Routines/Builds/Inner/CaseResult/CaseResultEditTest.tsx

```diff
@@ -45,13 +47,30 @@ const CaseResultEditTest = () => {
 
 	const {
 		caseResult,
+		caseResultsIssues,
 		mutateCaseResult,
+		mutateCaseResultIssues,
 	}: {
 		caseResult: TestrayCaseResult;
+		caseResultsIssues: TestrayCaseResultIssue[];
 		mutateCaseResult: KeyedMutator<any>;
```

Comment:
Add Type to KeyedMutator

```suggestion
		mutateCaseResult: KeyedMutator<TestrayCaseResult>;
```